src/graph.py

```diff
@@ -752,6 +750,104 @@ def perform_GO_enrichment_on_clusters(nxgraph, clustering):
     """
     pass
 
+
+###############################################################################
+            #######            Results           #######
+###############################################################################
+
+def summarize_grid_search(results, mode, top_n=-1): 
```

Comment:
Instead of -1, use False here as the default value. Then you can check later for `if (top_n)`.

Reply:
9ef8ea6871aedb9f9b0961c78e9becf366ea9216